src/pages/Dashboard/NoTeamRegistration.js

```diff
@@ -0,0 +1,347 @@
+import React, { useState } from 'react';
+import { useMutation, gql } from '@apollo/client';
+import { Box, Text, Button, Center, Input, FormControl } from '@chakra-ui/react';
+
+const NoTeamRegistration = props => {
+  const [currentPage, setPage] = useState(0);
+  const [teamName, setTeamName] = useState('');
+  const [teamAffiliation, setTeamAffiliation] = useState('');
+  const [inputList, setInputList] = useState([{ email: '' }]);
+
+  const incrementPage = async e => {
+    e.preventDefault();
+    if (currentPage < 2) {
+      setPage(currentPage + 1);
+    }
+  };
+
+  const decrementPage = async e => {
+    e.preventDefault();
+    if (currentPage > 0) {
+      setPage(currentPage - 1);
+    }
+  };
+
+  let pages = [
+    <InitialNoTeamPage incrementPage={incrementPage} />,
+    <CreateTeamForm
+      incrementPage={incrementPage}
+      decrementPage={decrementPage}
+      teamName={teamName}
+      teamAffiliation={teamAffiliation}
+      setTeamName={setTeamName}
+      setTeamAffiliation={setTeamAffiliation}
+      inputList={inputList}
+      setInputList={setInputList}
+    />,
+    <ConfirmTeamCreation
+      decrementPage={decrementPage}
+      teamName={teamName}
+      teamAffiliation={teamAffiliation}
+      userId={props.userId}
+      inputList={inputList}
+    />,
+  ];
+
+  return pages[currentPage];
+};
+
+const InitialNoTeamPage = props => {
+  return (
+    <Box fontFamily="Jost" w="full" h="full">
+      <Text fontSize="4xl" fontWeight="bold" marginLeft="5%" marginBottom={8}>
+        Welcome
+      </Text>
```

Comment:
Use `Heading` here and anywhere else the component in Figma is a heading

Also not sure where the `marginLeft` is coming from in the design, but I don't think we need it here (or anywhere else). If we did want a `marginLeft` on everything, you should add it to the outermost div :) 

```suggestion
      <Heading size="h1" as="h1" marginBottom={8}>
        Welcome
      </Heading>
```